README.md

```diff
@@ -960,39 +103,30 @@ const nextConfig = {
-
-  The simplest fix is to use the `autoConnect` client option and check if the client is created on the server side with a simple window object check, like this:
-
-  ```typescript
-  const client = new Ably.Realtime({ key: 'your-ably-api-key', autoConnect: typeof window !== 'undefined' });
-  ```
-
-  This will prevent the client from connecting to Ably servers if it is created on the server side, while not affecting your client side components.
+```typescript
+const client = new Ably.Realtime({ key: 'your-ably-api-key', autoConnect: typeof window !== 'undefined' });
+```
 
-- If you're using any React-based framework, you may be recreating the `Ably.Realtime` client instance on every component re-render. To avoid this, and to prevent potentially reaching the maximum connections limit on your account, move the client instantiation (`new Ably.Realtime`) outside of your components. You can find an example in our [React docs](./docs/react.md#Usage).
+#### Component re-renders
 
-- The connection limit error can be caused by the Hot Reloading mechanism of your development environment (called Fast Refresh in newer Next.js versions, or more generally, Hot Module Replacement - HMR). When you edit and save a file that contains a `new Ably.Realtime()` call in an environment that supports HMR (such as React, Vite, or Next.js apps), the file gets refreshed and creates a new `Ably.Realtime` client instance. However, the previous client remains in memory, unaware of the replacement, and stays connected to Ably's realtime systems. As a result, your connection count will keep increasing with each file edit as new clients are created. This only resets when you manually refresh the browser page, which closes all clients. This behavior applies to any development environment with an HMR mechanism implemented.
+Avoid creating the client inside React components. Instead, move the client instantiation outside of the component to prevent it from being recreated on every render.
 
-  The solution is simple: move the `new Ably.Realtime()` call to a separate file, such as `ably-client.js`, and export the client instance from there. This way, the client instance will only be recreated when you specifically make changes to the `ably-client.js` file, which should be far less frequent than changes in the rest of the codebase.
+#### Hot module replacement (HMR)
 
-## Contributing
+To avoid duplicate client instances caused by hot reloads, move the new `Ably.Realtime()` call into a separate file, for example, ably.js and export the client from there. This ensures a single shared instance is reused during development.
 
-For guidance on how to contribute to this project, see the [CONTRIBUTING.md](CONTRIBUTING.md).
+## Contribute
```

Comment:
I can see why you've made it active voice, but I think this should probably stay as 'contributing' since it's the file name and well known.